packages/shared/ReactFeatureFlags.js

```diff
@@ -78,7 +78,8 @@ export const enableLegacyFBSupport = false;
 // -----------------------------------------------------------------------------
 
 // Yield to the browser event loop and not just the scheduler event loop before passive effects.
-export const enableYieldingBeforePassive = __EXPERIMENTAL__;
+// Fix gated tests that fail with this flag enabled before turning it back on.
+export const enableYieldingBeforePassive = false;
```

Comment:
yes please, but just doing that won't enable the variant tests, you can do that by adding it here: 

https://github.com/facebook/react/blob/1e9ef39a8742889f8414c7df9c9e6ef463fe3d01/scripts/jest/setupTests.www.js#L13

Reply:
If you do that and tests fail though, just revert and I can look into enabling it

Reply:
> likely the same bug from your unit test since its asserting a function from an effect was called after resolving some Relay data

That might actually be an expected one where they previously relied on passive effects being flushed synchronously. Do they wrap the function that resolves the promise in `act()`?

Reply:
> yes please, but just doing that won't enable the variant tests, you can do that by adding it here:

Like this [`9948ef5` (#31857)](https://github.com/facebook/react/pull/31857/commits/9948ef542d3e3679d1c5e65b47b7aaff7d5e4633)?

Reply:
Seems like we didn't gate a bunch of tests on this. Reverting the variant change for now to unblock the sync.